Connected web3 provider at 10.195.3.52:22013
Date.now(): 1579031938905 startTime: 1579032000899 delay: 61997
Deploying smart contract from 0xDa0344835f667336920851580943f2B16de8E56D
TX HASH:
 0x5fa390f47599613c58474292861c4617455d9192c30202b79c5a3e0239d2a433
RECEIPT:
 {
  blockHash: '0xc49c5d4784908566b9ea937375e5fa779d1c04196f8ccbac576b727a98a7d637',
  blockNumber: 12,
  contractAddress: '0xf1520FeF044f4fad02e92F21a5a414DFD21E1EFB',
  cumulativeGasUsed: 218495,
  from: '0xda0344835f667336920851580943f2b16de8e56d',
  gasUsed: 218495,
  logs: [],
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  status: true,
  to: null,
  transactionHash: '0x5fa390f47599613c58474292861c4617455d9192c30202b79c5a3e0239d2a433',
  transactionIndex: 0
}
Contract at: 0xf1520FeF044f4fad02e92F21a5a414DFD21E1EFB

Evaluating  12800txs...

Sending TXS started at 1579032000977 ...
startingBlock: 20
... Sending TXS finished at 1579032099180
Total send duration: 98203
... Waiting for mining process to finish! ...
(node:21950) UnhandledPromiseRejectionWarning: Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
(node:21950) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:21950) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
ERROR:
 Error: Failed to check for transaction receipt:
{}
    at Object._fireError (/home/ubuntu/benchprezz/node_modules/web3-core-method/node_modules/web3-utils/src/index.js:56:17)
    at /home/ubuntu/benchprezz/node_modules/web3-core-method/src/index.js:260:23
    at runMicrotasks (<anonymous>)
    at runNextTicks (internal/process/task_queues.js:58:5)
    at listOnTimeout (internal/timers.js:501:9)
    at processTimers (internal/timers.js:475:7)
========================================================
========================================================

/home/ubuntu/benchprezz/benchmark/tpsUnsigned.js:113
                throw 'Blockchain capabilities EXCEEDED - ABORTING test run!';
                ^
Blockchain capabilities EXCEEDED - ABORTING test run!
